features/include/pcl/features/impl/fpfh.hpp

```diff
@@ -73,7 +73,7 @@ pcl::FPFHEstimation<PointInT, PointNT, PointOutT>::computePointSPFHSignature (
   float hist_incr = 100.0f / static_cast<float>(indices.size () - 1);
 
   // Iterate over all the points in the neighborhood
-  for (const int &index : indices)
+  for (const auto &index : indices)
```

Comment:
I think auto here decreases readability for the same reasons.

Reply:
`auto` here enables changing the indices to `std::size_t` or `std::ptrdiff_t` later. That change will be quite difficult without minor changes like this.

Reply:
@kunaltyagi That's a valid point. But I would argue that the change won't be that difficult with vim + regular expression search and replace. So I would rather prefer keeping good readability.